SCRAM/GMake/Makefile.rules

```diff
@@ -366,7 +366,7 @@ define copy_build_product
 endef
 
 define copy_lib_to_main_productstore
-  $(call install_name_tool,$(@F),$@) && $(call copy_build_product,$(1))
+  $(call install_name_tool,$(LOCALTOP)/$(1)/$(@F),$@) && $(call copy_build_product,$(1))
 endef
```

Comment:
may be we should make it a configurable i.e. only set id to full path if project configure it via its project file (Self.xml or config/BuildFile.xml)